### DISCUSSION TITLE
Passaggio da raspberry a mini pc e add-on che non si avviano

### DISCUSSION THREAD

**Discussion by Moxser** [indomus/forum#2818] [HUB: Home Assistant]
Ciao a tutti.
Ho deciso dopo qualche anno di passare da raspberry a mini pc sempre con l'installazione di HassOS perché per le mie conoscenze è più che sufficiente e mi ci trovo molto bene.
Ho seguito la vostra guida e tutto è andato a buon fine, ho ripristinato il backup tutto funziona ma gli add-on non partono dopo il ripristino.
Quindi SSH, cloudflare, File editor, Home assistant google drive backup, Mosquitto broker, samba, non riesco ad avviarli.
L'unica cosa che mi viene in mente sarebbe quella di fare un copia e incolla su un file txt delle configurazioni dei diversi add-on e copiarle uno ad uno dopo l'avvio del mini pc perchè sembra che il ripristino non se le porti con se dopo il cambio di hardware.
Ora ho spento il nuovo mini pc e riacceso il mio affidabile raspberry che naturalmente funziona alla grande.
Qualcuno mi potrebbe dare qualche dritta per risolvere il problema?

**Comment by snakuzzo** [comment]
Dopo aver ripristinato il backup, prova a fare un riavvio del sistema.
Detto questo, quando qualcosa non va devi imparare a guardare i log...e parlo, nel tuo caso, dei log dei singoli addon e dei log del supervisor.

**Reply by 50494554524F** [reply]
è un problema che ho riscontrato anche io, ho risolto così...
- se attivo, deseleziona avvio automatico
- arresta ha, non fare riavvio
- riaccendi
- riseleziona avvio automatico dove serve
- riavvia e verifica se parte tutto
ora dovrebbe andare...

**Reply by Moxser** [reply]
Ho provato il tuo sistema e ora tutto funziona perfettamente, disattivato l'avvio degli add on in automatico, spento, riacceso, riattivato l'avvio automatico degli add on, riavviato e tutto ok. Per cloudflare vedendo i log come consigliato da snakuzzo ho inserito nel configurator.yaml i trusted proxies mancanti e tutto è andato a buon fine. 
Ora sono definitivamente passato a un mini Pc. Grazie e tutti per l'aiuto.
